src/main/java/ch/admin/bag/covidcertificate/service/NotificationService.java

```diff
@@ -50,10 +51,13 @@
     @CacheEvict(value = NOTIFICATIONS_CACHE_NAME, allEntries = true)
     public void writeNotifications(List<NotificationDto> notifications) {
         log.info("Write notifications");
-        ObjectMapper mapper = new ObjectMapper();
+
+        if ((!this.notificationRepository.findAll().isEmpty())) {
```

Comment:
I recommand the use of '.count()', for this purpose 'findAll()' is a heavy operation.